docs/configurations.md

```diff
@@ -101,123 +127,80 @@ The following describes the configurations available for each storage.
+</div>
+</div>
 
-```properties
-scalar.db.contact_points=jdbc:sqlite:<YOUR_DB>.sqlite3?busy_timeout=10000
-```
+##### Multi-storage support
 
-Unlike other JDBC databases, [SQLite3 does not fully support concurrent access](https://www.sqlite.org/lang_transaction.html).
-To avoid frequent errors caused internally by [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy), we recommend setting a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
+ScalarDB supports using multiple storage implementations simultaneously. You can use multiple storages by specifying `multi-storage` as the value for the `scalar.db.storage` property.
 
-## ScalarDB Server configurations
+For details about using multiple storages, see [Multi-Storage Transactions](multi-storage-transactions.md).
 
-[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB.
-This section explains ScalarDB Server configurations.
+### Use Consensus Commit through ScalarDB Cluster
 
-In addition to [transaction manager configurations](#transaction-manager-configurations) and [other configurations](#other-configurations), the following configurations are available for ScalarDB Server:
+[ScalarDB Cluster (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/) is a component that provides a gRPC interface to ScalarDB.
 
-| Name                                              | Description                                                                                      | Default                |
-|---------------------------------------------------|--------------------------------------------------------------------------------------------------|------------------------|
-| `scalar.db.server.port`                           | Port number for ScalarDB Server.                                                                 | `60051`                |
-| `scalar.db.server.prometheus_exporter_port`       | Prometheus exporter port. Prometheus exporter will not be started if a negative number is given. | `8080`                 |
-| `scalar.db.server.grpc.max_inbound_message_size`  | The maximum message size allowed to be received.                                                 | The gRPC default value |
-| `scalar.db.server.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.                                             | The gRPC default value |
-| `scalar.db.server.decommissioning_duration_secs`  | The decommissioning duration in seconds.                                                         | `30`                   |
+For configurations available for the gRPC transaction manager for ScalarDB Cluster, see the ScalarDB Cluster [client configurations (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/developer-guide-for-scalardb-cluster-with-java-api/#client-configurations).
```

Comment:
I think the following is better. Please correct it for the English perspective.
```suggestion
For client configurations for ScalarDB Cluster, see the ScalarDB Cluster [client configurations (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/developer-guide-for-scalardb-cluster-with-java-api/#client-configurations).
```

Reply:
Thank you for your suggestion! I've simplified the sentence.